src/ssmDocument/commands/updateDocumentVersion.ts

```diff
@@ -0,0 +1,118 @@
+                } else {
+                    vscode.window.showInformationMessage(
+                        localize(
+                            'AWS.message.info.ssmDocument.updateDocumentVersion.success',
+                            'Updated document {0} default version to {1} successfully',
+                            node.documentName,
+                            documentVersion
+                        )
+                    )
+                }
+            }
+        } else {
+            result = 'Failed'
+            vscode.window.showErrorMessage(
+                localize(
+                    'AWS.message.error.ssmDocument.updateDocumentVersion.does_not_own',
+                    'Could not update document {0} default version. The current account does not have ownership of this document.',
+                    node.documentName
+                )
+            )
+        }
+    } catch (err) {
+        result = 'Failed'
+        const error = err as Error
+        vscode.window.showErrorMessage(
+            localize(
+                'AWS.message.error.ssmDocument.updateDocumentVersion.could_not_update_version',
+                'Could not update document {0} default version. Please check logs for more details.',
+                node.documentName
+            )
+        )
+        logger.error('Error on updating document version: %0', error)
+    } finally {
+        telemetry.recordSsmUpdateDocumentVersion({ result: result })
+    }
+}
+
+async function promptUserforDocumentVersion(versions: SSM.Types.DocumentVersionInfo[]): Promise<string | undefined> {
+    // Prompt user to pick document version
+    const quickPickItems: vscode.QuickPickItem[] = []
+    versions.forEach(version => {
+        if (version.DocumentVersion) {
+            quickPickItems.push({
+                label: version.DocumentVersion,
+                description: `${version.IsDefaultVersion ? 'Default' : ''}`,
+            })
+        }
+    })
+
+    if (quickPickItems.length > 1) {
+        const versionPick = picker.createQuickPick({
+            options: {
+                ignoreFocusOut: true,
+                title: localize(
+                    'AWS.message.prompt.selectSsmDocumentVersion.placeholder',
+                    'Select a document version to download'
+                ),
+            },
+            items: quickPickItems,
+        })
+
+        const versionChoices = await picker.promptUser({
+            picker: versionPick,
+        })
+
+        const versionSelection = picker.verifySinglePickerOutput(versionChoices)
+
+        // User pressed escape and didn't select a template
+        return versionSelection?.label
+    } else {
+        vscode.window.showInformationMessage(
+            localize(
+                'AWS.message.info.ssmDocument.updateDocumentVersion.no_other_versions',
+                'Selected document has only one version. Unable to change default version.'
+            )
+        )
+        return undefined
```

Comment:
Is this the correct statement? Does this function directly tie to changing the default version?

Also, should this return `undefined` if there's one item, or should this auto return the individual item?

Reply:
We don't allow users to change the default version if there's only 1 default version. We don't want a version returned in the case of one item as not to proceed with changing the default version. Returning undefined here will allow this function to cancel.